test/src/main/java/io/strimzi/test/k8s/cmdClient/BaseCmdKubeClient.java

```diff
@@ -342,6 +351,18 @@ public String getResourceAsYaml(String resourceType, String resourceName) {
         return Exec.exec(namespacedCommand("get", resourceType, resourceName, "-o", "yaml")).out();
     }
 
+    @Override
+    public void createResourceAndApply(String template, Map<String, String> params) {
+        List<String> cmd = namespacedCommand("process", template, "-l", "app=" + template, "-o", "yaml");
+        for (Map.Entry<String, String> entry : params.entrySet()) {
+            cmd.add("-p");
+            cmd.add(entry.getKey() + "=" + entry.getValue());
+        }
+
+        String yaml = Exec.exec(cmd).out();
+        createContent(yaml);
```

Comment:
why we cannot `apply` (instead of `create`) the created yaml content?

Reply:
when I tried that with `applyContent` (because if you use only `apply` it needs file) I got really strange annotation that includes CR in JSON. I will try that again and I'll see.

Reply:
I tried that right now and it seems to work, so thank you for nit :)